### PR TITLE
[css-writing-mode] Move sideways-* tests to L4>

### PR DESCRIPTION
**Pull request by fantasai**

### CODE REVIEW

File: css/css-writing-modes/inline-block-alignment-003.xht [hunk 1]
```diff
@@ -14,7 +14,7 @@
     {
       color: orange;
       font: 60px/1 Ahem; /* computes to 60px/60px */
-      writing-mode: vertical-lr;
+      xwriting-mode: vertical-lr;
```
Comment (kojiishi) [inline css/css-writing-modes/inline-block-alignment-003.xht]: Is this typo?

**Review by kojiishi — APPROVED** [review]
Looks great, except one probably a typo, once it's fixed, this is good to land.

File: css/css-writing-modes/inline-block-alignment-003.xht (hunk 1 above)
Comment (fantasai) [reply css/css-writing-modes/inline-block-alignment-003.xht]: Yes :(

**Review by fantasai — COMMENTED** [review]
Fixed, thanks!

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/25687)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/351184183)

**Pull request merged**